test/python/circuit/test_controlled_gate.py

```diff
@@ -1443,9 +1449,8 @@ def test_single_controlled_rotation_gates(self, gate, cgate):
         cqc = QuantumCircuit(self.num_ctrl + self.num_target)
         cqc.append(cgate, cqc.qregs[0])
         dag = circuit_to_dag(cqc)
-        with self.assertWarns(DeprecationWarning):
-            unroller = Unroller(["u", "cx"])
-        uqc = dag_to_circuit(unroller.run(dag))
+        basis_translator = BasisTranslator(std_eqlib, ["u", "cx", "ccu1", "ccrx", "ccry", "ccrz"])
```

Comment:
I think this might also not test the right thing anymore: by adding the operations such as `ccrz` to the basis translator, doesn't the basis translator just do nothing in the `cgrz` case? If the number of operations in the `u cx` basis increases using the basis translator instead of the unroller, we could change the test threshold (and might need to open an issue to investigate, if there's a signficant increase)

Reply:
Hmmm, I was a bit confused here and that does make sense, but I am not currently sure how to change the test threshold. When I was attempting to convert this from Unroller to BasisTranslator, not including the operations such as `ccrz` would cause a similar `TranspilerError` like above. 

For instance, if we run `basis_translator = BasisTranslator(std_eqlib, ["u", "cx"])` instead, then the output for `test_single_controlled_rotation_gates_1` would be 
` qiskit.transpiler.exceptions.TranspilerError: "Unable to translate the operations in the circuit: ['ccu1'] to the backend's (or manually specified) target basis: ['snapshot', 'cx', 'delay', 'measure', 'barrier', 'u', 'reset']. This likely means the target basis is not universal or there are additional equivalence rules needed in the EquivalenceLibrary being used. For more details on this error see: https://docs.quantum-computing.ibm.com/api/qiskit/qiskit.transpiler.passes.BasisTranslator#translation-errors"`